src/test/java/com/epam/jamp2/model/CalculationCommandTest.java

```diff
@@ -0,0 +1,85 @@
+package com.epam.jamp2.model;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+import static org.hamcrest.Matchers.*;
+
+//@RunWith(SpringJUnit4ClassRunner.class)
+public class CalculationCommandTest {
+
+	@Test
+	public void testCommand1() throws CommandFormatException {
+		String cmdStr = "usd100.5+cny5.0=chD";
+		CalculationCommand cmd = CalculationCommand.parseFromString(cmdStr);
+		Value leftOper = cmd.getLeftOperand();
+		Value rightOper = cmd.getRightOperand();
+		Operation oper = cmd.getOperation();
+		Optional<String> ccy = cmd.getResultCurrencyCode();
+		Assert.assertThat(leftOper.getCurrencyCode().get(), equalToIgnoringCase("USD"));
+		Assert.assertThat(rightOper.getCurrencyCode().get(), equalToIgnoringCase("CNY"));
+		Assert.assertThat(leftOper.getValue(), equalTo(new BigDecimal("100.5")));
+		Assert.assertThat(rightOper.getValue(), equalTo(new BigDecimal("5.0")));
+		Assert.assertThat(oper, equalTo(Operation.ADD));
+		Assert.assertThat(ccy.get(), equalToIgnoringCase("CHD"));
+	}
+	
+	@Test
+	public void testCommand2() throws CommandFormatException {
+		String cmdStr = "usd100.0/cny5.1=chD";

```

Comment:
For positive scenarios where only operation is different parametrized tests might be fitting best. They would reduce your code volume significantly, while preserving test coverage level.